gtsfm/densify/mvs_base.py

```diff
@@ -14,10 +14,22 @@
 from gtsfm.common.gtsfm_data import GtsfmData
 from gtsfm.evaluation.metrics import GtsfmMetricsGroup
 
+from gtsfm.ui.gtsfm_process import GTSFMProcess, UiMetadata
 
-class MVSBase(metaclass=abc.ABCMeta):
+
+class MVSBase(GTSFMProcess):
     """Base class for all multi-view stereo implementations."""
 
+    def get_ui_metadata() -> UiMetadata:
+        """Returns data needed to display this process in the process graph."""
+
+        return UiMetadata(
```

Comment:
`UiMetadata` seems a little vague, but maybe ok. ProcessGraphMetadata? @akshay-krishnan thoughts?

Reply:
Yeah, ProcessUiMetadata or ProcessGraphMetadata. but optional, im okay with UiMetadata too.

Reply:
I will keep UiMetadata then. I believe the improved docstring @johnwlambert [requested](https://github.com/borglab/gtsfm/pull/540#discussion_r947427117) will clarify things as well.